apis/kustomize/kustomize_types.go

```diff
@@ -93,7 +93,7 @@ type Patch struct {
 
 	// Target points to the resources that the patch document should be applied to.
 	// +optional
-	Target Selector `json:"target,omitempty"`
+	Target *Selector `json:"target,omitempty"`
```

Comment:
It is not breaking except for consumption from the Go-side as the YAML behavior is leading for the APIs.

Reply:
@darkowlzz this will be released as apis/kustomize v1 as for  KC and HC release notes there is no need for that, this has no impact on users expect for the bug fix.

Reply:
Sure they will, Kustomization v1 API will come with other breaking change besides this. I'll make sure this is mentioned in the API changelog.